Create agenda & meeting minutes for 2-25-26 TAC meeting (#133)
PQCA/TAC: meeting-minutes/2026/2026-02-25-TAC.md

- The call for PQCA mentorship program mentors remains open for the Summer 2026 session (June–August).

---

### **OQS Metrics and Blog Follow-Ups**
Norman confirmed that the OQS metrics discussion remains unchanged. Brian suggested continuing forward with periodic updates rather than waiting for further refinements.

The CBOMkit pipeline integration blog post remains paused due to the author’s absence.

---

### **OpenSSF Baselines Discussion**
Hart introduced the topic of [**OpenSSF Baselines**](https://baseline.openssf.org/), describing them as a structured security checklist that can help projects demonstrate adherence to best practices. While PQCA projects are already security-focused, certification could serve as:
- A structured self-assessment
- An external signaling mechanism for enterprise users

The TAC agreed that each project TSC (OQS, CBOMkit, PQCP) should evaluate:
- The value of pursuing OpenSSF baseline compliance
- The appropriate level to target
- Whether the effort would provide strategic differentiation

Hart offered to collect feedback and relay it to OpenSSF leadership.

---

### **Project Updates**
- OQS (Norman)
  - 

> Norman reported ongoing planning for the next OQS release. The team discussed coordinating mlk-native update timing, especially in light of increased release frequency from CodePackage.

Reviewer: ```suggestion
Norman reported ongoing planning for the next OQS release. The team discussed coordinating mlkem-native update timing, especially in light of increased release frequency from PQCP.
```

- PQCP (Matthias)
  - 
Matthias shared updates including:

Upcoming talk about mlkem-native at RWC 2026 on March 9, 2026.

**mlkem-native & mldsa-native**
- Progress in our experimental Armv8.1-M/MVE backend (see pq-code-package/mlkem-native#1524)

**mlkem-native:**
- Porting HOL-Light constant-time and memory safety proofs from s2n-bignum (and filling in gaps)
- Wrapping up conversion of x86_64 compression intrinsics to assembly + HOL-Light correctness proofs
- x86_64 AVX2 Keccak implementation with HOL-Light correctness proof contributed by @manastasova - under review

**mldsa-native:**
- (Slow) progress towards HOL-Light correctness proofs of native code (x86: 3/17; AArch64: 4/17 -- some still under review)
- Added Cortex-M33 baremetal tests - considering an Armv7/8-M backend

- CBOMkit (Andreas)
  - 
Andreas reported:
- Golang support added to CBOMkit
- Version upgrade to 2.2
- Improved support for Go crypto libraries
- Potential future language support (C/C++), pending contributor interest

> 

Reviewer 2: ```suggestion
- OQS (Norman)
  - There is ongoing planning for the next OQS release. 
  - The team discussed coordinating mlkem-native update timing, especially in light of increased release frequency from PQCP.
- PQCP (Matthias)
  - Upcoming talk about mlkem-native at RWC 2026 on March 9, 2026.
  - Progress in our experimental Armv8.1-M/MVE backend (see pq-code-package/mlkem-native#1524)
  - mlkem-native
    - Porting HOL-Light constant-time and memory safety proofs from s2n-bignum (and filling in gaps)
    - Wrapping up conversion of x86_64 compression intrinsics to assembly + HOL-Light correctness proofs
    - x86_64 AVX2 Keccak implementation with HOL-Light correctness proof contributed by @manastasova - under review
  - mldsa-native:
    - (Slow) progress towards HOL-Light correctness proofs of native code (x86: 3/17; AArch64: 4/17 -- some still under review)
    - Added Cortex-M33 baremetal tests - considering an Armv7/8-M backend
- CBOMkit (Andreas)
  - Golang support added to CBOMkit
  - Version upgrade to 2.2
  - Improved support for Go crypto libraries
  - Potential future language support (C/C++), pending contributor interest
```

The markdown indentations were bonkers, so cleaned it up a bit

---

## Voting Representative Attendance (_Alphabetical by 1st name_)
### Premier Member Representatives

* [ ] Brian Jarvis, AWS [TAC Chair]
* [ ] Michael Maximilien, IBM
* [ ] Norman Ashley, Cisco (OQS Rep)
* [ ] Sophie Schmieg, Google

 ### Project Representatives
* [ ] Matthias Kannwischer, TCR
* [ ] Hanno Becker, AWS (PQCP)
* [ ] Andreas Schade, IBM (CBOMkit)

## Non-Voting Representative Attendance

### LF Staff 
* [ ] Christina Harter 
* [ ] Hart Montgomery 
* [ ] Min Yu

### Other Attendees
* [ ] Aditya Koranga, NgKore \[TAC Vice Chair\]
* [ ] *Enter other attendees*

---
# Meeting Agenda
- Christina: TAC Chair and Technical Community Representative (TCR) nominations now open
- Christina: Reminder: Call for Mentors for PQCA Summer Mentorship Program

> - Christina: OQS Fall 2025 mentee blog post - currently under review; plan to publish this week

Reviewer 2: ```suggestion
Follow ups:
 - OQS metric
 - CBOMkit pipeline integration blog post

New items:
- Christina: TAC Chair and Technical Community Representative (TCR) nominations now open
- Christina: Reminder: Call for Mentors for PQCA Summer Mentorship Program
- Christina: OQS Fall 2025 mentee blog post - currently under review; plan to publish this week
- OpenSSF baseline for PQCA projects

```
